src/main/resources/lang/default.lang

```diff
@@ -2555,6 +2555,92 @@ input keys:
 	sneak: sneak key, sneaking key
 	sprint: sprint key, sprinting key
 
+# -- Statistics --
+statistics:
```

Comment:
i know it's likely in the service of avoiding conflicts, but forcing `statistic` twice in the syntax gets very clunky. Do you think it's feasible to omit the word statistic from these names?